docs/UserMenu.md

```diff
@@ -0,0 +1,61 @@
+# User Menu
+
+The `<UserMenu>` is an Avatar that opens a Menu when clicked. It is typically used in the top-right corner of an application and indicates who is logged in.
+
+<div style="align-items: center; display:flex; justify-content: space-around">
+
+<img width="40%" alt="UserMenu Avatar" src="./images/userMenuAvatar.png">
+<img width="40%" alt="UserMenu Opened" src="./images/userMenuOpened.png">
+
+</div>
+
+## Usage
+
+```tsx
+import { UserMenu, InfoListItemProps, wrapIcon } from '@pxblue/react-native-components';
+import MatIcon from 'react-native-vector-icons/MaterialIcons';
+import * as Colors from '@pxblue/colors';
+...
+const VpnKeyIcon = wrapIcon({ IconClass: MatIcon, name: 'vpn-key', flip: false });
+const SettingsIcon = wrapIcon({ IconClass: MatIcon, name: 'settings', flip: false });
+const ExitToAppIcon = wrapIcon({ IconClass: MatIcon, name: 'exit-to-app', flip: false });
+
+const menuItems: InfoListItemProps[] = [
+    { title: 'Change Password', IconClass: VpnKeyIcon, onPress: (): void => {} },
+    { title: 'Preferences', IconClass: SettingsIcon, onPress: (): void => {} },
+    { title: 'Log Out', IconClass: ExitToAppIcon, onPress: (): void => {} },
+];
+
+<UserMenu
+    menuTitle={'John Smith'}
+    menuSubtitle={'j.smith@example.com'}
+    menuItems={menuItems}
+    avatar={<Avatar.Text label={'PX'} size={40} color={Colors.white[50]} />}
+/>
+```
```

Comment:
This usage example is really big...not sure what we're doing for the others, but maybe we can simplify this.
```suggestion
```tsx
import { UserMenu } from '@pxblue/react-native-components';
...
<UserMenu
    menuTitle={'John Smith'}
    menuSubtitle={'j.smith@example.com'}
    menuItems={[
        { title: 'Change Password', IconClass: VpnKeyIcon },
        { title: 'Preferences', IconClass: SettingsIcon },
        { title: 'Log Out', IconClass: ExitToAppIcon },
    ]}
    avatar={<Avatar.Text label={'PX'} size={40} />}
/>
```
```